### PR TITLE
Changes based on munificent's comments

### PR DESCRIPTION
**Pull request by ppvk**
Sorry for all the whitespace trimming, my editor automatically does that. :/

The important stuff:
- `WrenErrorFn` is removed in favor of reusing `WrenWriteFn`.
-  `config->errorFn` is now set to `defaultErrorFn` during `wrenInitConfiguration`
- `defaultErrorFn` simply wraps `fprintf`removing the need for a conditional in `wrenPrintError`

### CODE REVIEW

**Comment by preble** [comment]
Thanks @ppvk! I'll have a closer look this evening. Until then, can you look into removing the whitespace changes? A number of graphical git apps let you stage only the lines you intended to change. If you're on a Mac, check out [GitUp](http://gitup.co). The trouble with whitespace changes is that they clutter up the git blame, which makes it difficult to understand why changes were made.

**Comment by ppvk** [comment]
Will do!

File: src/vm/wren_vm.c [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #include <stdarg.h>
 #include <string.h>
-#include <stdio.h>
```
Comment (preble) [inline src/vm/wren_vm.c]: I had moved this from `#if WREN_DEBUG_TRACE_MEMORY || WREN_DEBUG_TRACE_GC`, below, so it should go back there. See: https://github.com/ppvk/wren/blob/master/src/vm/wren_vm.c

**Comment by preble** [comment]
Looks good! Merging!

**Pull request merged**